src/main/java/org/cloudburstmc/server/block/behavior/ReedsBlockBehavior.java

```diff
@@ -116,7 +116,7 @@ public boolean place(ItemStack item, Block block, Block target, Direction face,
 
         if (downType == BlockTypes.REEDS) {
             return placeBlock(block, CloudBlockRegistry.get().getBlock(BlockTypes.REEDS));
-        } else if (downType == BlockTypes.GRASS || downType == BlockTypes.DIRT || downType == BlockTypes.SAND) {
+        } else if (downType == BlockTypes.GRASS || downType == BlockTypes.DIRT || downType == BlockTypes.SAND || downType == BlockTypes.PODZOL || downType == BlockTypes.MYCELIUM) {
```

Comment:
it could be useful to introduce some block category but not sure about the correct name

Reply:
https://github.com/CloudburstMC/API/pull/14